flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontend.java

```diff
@@ -86,6 +87,18 @@ static CliExecutor createExecutor(CommandLine commandLine) throws Exception {
         // Global pipeline configuration
         Configuration globalPipelineConfig = getGlobalConfig(commandLine);
 
+        // Create pipeline composer
+        boolean useMiniCluster = commandLine.hasOption(CliFrontendOptions.USE_MINI_CLUSTER);
```

Comment:
How could the MiniCluster run without specifying `FLINK_HOME`? We don't package `flink-dist` into our distribution, so I assume that the MiniCluster cannot even be started without specifying the place where Flink related classes are.